README.md

```diff
@@ -107,10 +98,26 @@ metadata.add_table({
         {"name": "name", "data_type": "text"}
     ],
 })
+```
 
+Then, to export this object to a JSON file, you can run the following:
+
+```
 metadata.to_json("your_metadata.json")
 ```
 
+## Validating JSON Files
+
+The core functionality of this library is to validate JSON files. The following code will load 
```

Comment:
I think that this could be rephrased. Since we already presented a feature above, it feels weird to say that this is "the core functionality"